.github/workflows/reopen_jira_issue.yml

```diff
@@ -0,0 +1,30 @@
+name: Reopen JIRA Issue
+on:
+    issues:
+        types:
+            - reopened
+
+jobs:
+    reopen_jira_issue:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - uses: actions/setup-python@v5
+              with:
+                python-version: '3.9'
+                cache: 'pip'
+            - name: install dependencies
+              env:
+                SCRIPT_DIR: ${{ github.workspace }}/.github/workflows/scripts
+              run: pip install -r ${SCRIPT_DIR}/requirements.txt
+            - name: Reopen JIRA Task
+              env:
+                SCRIPT_DIR: ${{ github.workspace }}/.github/workflows/scripts
+                JIRA_TOKEN: ${{ secrets.JIRA_TOKEN  }}
+                JIRA_USER: ${{ secrets.JIRA_USER  }}
+                ISSUE_URL: ${{ github.event.issue.html_url }}
+                ISSUE_TITLE: ${{ github.event.issue.title }}
+                ISSUE_BODY: ${{ github.event.issue.body }}
+                GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+                ISSUE_LABELS: ${{ join(github.event.issue.labels.*.name) }}
+              run: python ${SCRIPT_DIR}/jira_helper.py REOPEN_ISSUE --verbose
```

Comment:
Since all these workflows are the same thing but with slightly different environment variables and a different command, can we pull this out into a custom composite action?

Composite Actions: https://docs.github.com/en/actions/creating-actions/creating-a-composite-action

Reply:
Sounds like a nice imporvement... on the backlog :)